pkg/apiserver/cmapis.go

```diff
@@ -80,7 +79,6 @@ func (s *CustomMetricsAdapterServer) cmAPI(groupInfo *genericapiserver.APIGroupI
 			Convertor:       groupInfo.Scheme,
 			UnsafeConvertor: runtime.UnsafeObjectConvertor(groupInfo.Scheme),
 			Typer:           groupInfo.Scheme,
-			Linker:          runtime.SelfLinker(meta.NewAccessor()),
```

Comment:
The linker has been removed in favor of a `Namer`, so instead of removing it entirely could you perhaps set the namers where needed in the instantiations of  APIGroupVersion and ContextBasedNaming as in https://github.com/kubernetes/kubernetes/pull/107527/commits/0ad588b27b6845d9ece955bfdde0e2863b8e1394?

Reply:
sure, on it!

Reply:
done! could you take another look?

Reply:
looks good to me :+1: